backend/api/game.go

```diff
@@ -31,10 +31,33 @@ func gameInfoHandler(c echo.Context) error {
 	if err != nil {
 		return c.JSON(http.StatusInternalServerError, "Database find game ID error")
 	}
-
 	return c.JSON(http.StatusOK, game)
 }
 
+func getGamesWithTags(c echo.Context) error {
+	// connect the db
+	_db, err := db.NewDatabaseFromEnv()
+	if err != nil {
+		log.WithError(err).Error("Database connection error in API getGamesWithTags")
+		return c.JSON(http.StatusInternalServerError, "Database connection error")
+	}
+	// // grab tags from context
+	// rawTags := c.FormValue(TAGS)
+
+	// // split string into array
+	// tags := strings.Split(rawTags, ",")
```

Comment:
Was this intended or should this commented-out code be removed?

Reply:
I didn't want to throw this code because this is not my code that's why